evap/development/fixtures/test_data.json

```diff
@@ -133247,6 +133247,16 @@
     "html_content": "<i>(English version below)</i><br /><br /><br />\r\n\r\n\r\nHallo {{ user.first_name }},<br /><br />\r\n\r\nes gibt noch nicht überprüfte Textantworten für eine oder mehrere Evaluierungen, bei denen der Evaluierungszeitraum abgelaufen ist und nicht mehr auf Notenveröffentlichungen gewartet werden muss. Bitte überprüfe die Textantworten für diese Evaluierungen möglichst bald:\r\n<ul>{% for evaluation, url in evaluation_url_tuples %}\r\n<li><a href='{{ url }}'>{{ evaluation.full_name_de }}</a></li>\r\n{% endfor %}</ul><br /><br />\r\n\r\n<i>(Dies ist eine automatisch versendete E-Mail.)</i><br /><br />\r\n\r\n<hr><br /><br />\r\n\r\nDear {{ user.first_name }},<br /><br />\r\n\r\nthere are text answers not yet reviewed for one or more evaluations where the evaluation period has ended and there is no need to wait for grade publishing. Please review the text answers for these evaluations as soon as possible:\r\n<ul>{% for evaluation, url in evaluation_url_tuples %}\r\n<li><a href='{{ url }}'>{{ evaluation.full_name_en }}</a></li>\r\n{% endfor %}</ul><br /><br />\r\n\r\n<i>(This is an automated message.)</i>"
   }
 },
+{
+  "model": "evaluation.emailtemplate",
+  "pk": 10,
+  "fields": {
+    "name": "Grade Reminder",
+    "subject": "[EvaP] Notenveröffentlichung für {{semester.name_de}} / Grade publishing for {{semester.name_en}}",
+    "plain_content": "(English version below)\r\n\r\nLiebes Studienreferat,\r\n\r\nfür die folgenden Veranstaltungen im {{semester.name_de}} wurden bisher keine Notendokumente auf EvaP hochgeladen. Bitte prüfen Sie, ob die Noten bereits vorliegen und die Evaluierungsergebnisse veröffentlicht werden können.\r\n\r\n{% for responsible, courses in responsibles_and_courses_without_final_grades %}\r\n{{ responsible }}\r\n{% for course in courses %}\r\n- {{ course.name_de }} ({{ course.type }})\r\n{% endfor %}\r\n\r\n{% endfor %}\r\n\r\nBei Fragen und Rückmeldungen können Sie sich jederzeit gerne an das Evaluierungsteam wenden ({{ contact_email }}).\r\n\r\nFreundliche Grüße,\r\ndas Evaluierungsteam\r\n\r\n(Dies ist eine automatisch versendete E-Mail.)\r\n\r\n\r\n--\r\n\r\n\r\nDear office for student affairs,\r\n\r\nfor the following courses in {{semester.name_en}}, no grade documents have yet been uploaded on EvaP. Please check whether the grades are already available and whether the evaluation results can be published.\r\n\r\n{% for responsible, courses in responsibles_and_courses_without_final_grades %}\r\n{{ responsible }}\r\n{% for course in courses %}\r\n- {{ course.name_en }} ({{ course.type }})\r\n{% endfor %}\r\n\r\n{% endfor %}\r\n\r\nIf you have any questions or feedback, please contact the evaluation team ({{ contact_email }}).\r\n\r\nKind regards,\r\nthe Evaluation Team\r\n\r\n(This is an automated message.)",
```

Comment:
Not 100% sure, but shouldn't it be "no grade documents have been uploaded to EvaP **yet**" instead of "no grade documents have **yet** been uploaded to EvaP"?